src/reference/sectionC_interpretation.inc

```diff
@@ -400,23 +403,23 @@ Interpretation of ``map_variables`` elements
 
     #. If the :code:`interface` attribute is absent, then the variable has no interface.
 
-7.  The applicable interfaces for variables *A* and *B* in components *AA* and *BB* respectively SHALL be defined as follows.
+7.  The "applicable interfaces" for variables *A* and *B* in components *AA* and *BB* respectively SHALL be defined as follows:
 
     #. When component *AA* is in the :ref:`sibling set<specC_sibling_set>` of component *BB* (and vice versa), the applicable interface for both variables *A* and *B* SHALL be the public interface.
 
     #. When component *AA* is in the :ref:`encapsulated set<specC_encapsulation_set>` of component *BB* the applicable interface for variable *A* SHALL be the public interface, and the applicable interface for variable *B* SHALL be the private interface.
 
     #. When component *BB* is in the :ref:`encapsulated set<specC_encapsulation_set>` of component *AA* the applicable interface for variable *B* SHALL be the public interface, and the applicable interface for variable *A* SHALL be the private interface.
 
-    #. It is noted, for the avoidance of doubt, that any other encapsulation relationship between components *AA* and *BB* will not permit connections, meaning that no interface is available between the variables *A* and *B*.
+    It is noted, for the avoidance of doubt, that if components *AA* and *BB* are in each other's :ref:`hidden set<specC_hidden_set>`, there are no applicable interfaces for the variables *A* and *B*.
 
-8.  CellML models MUST only contain :code:`map_variables` elements where the interface of variable *A* and the interface of variable *B* are applicable interfaces.
+8.  CellML models MUST only contain :code:`map_variables` elements where the applicable interfaces for both variables are available.
```

Comment:
Shouldn't this read "... where the applicable interface for both variable is available"? A variable can only have one applicable interface, not several of them...?

Reply:
for _both_ variables, so the applicable interface for A must be available, and the applicable interface for B must be available

Reply:
Ok, I stand corrected. :)